src/client/components/app-header/app-header.vue

```diff
@@ -1,18 +1,20 @@
 <template>
-  <md-toolbar class="md-primary">
-    <nuxt-link to="/">
-      <h1 class="md-title">Keukentafeltool</h1>
-    </nuxt-link>
-    <div class="md-toolbar-section-end">
-      <md-button
-        class="md-icon-button"
-        @click="$emit('refresh')"
-      >
-        <md-icon>refresh</md-icon>
-      </md-button>
-      <fullscreen-button/>
-    </div>
-  </md-toolbar>
+  <header role="banner">
+    <md-toolbar class="md-primary">
+      <nuxt-link to="/">
+        <h1 class="md-title">Keukentafeltool</h1>
+      </nuxt-link>
+      <div class="md-toolbar-section-end">
+        <md-button
+          class="md-icon-button"
+          @click="$emit('refresh')"
```

Comment:
would the 'replay' icon be a better choice? for ltr readers the arrow go's back and not forward like the 'refresh' one does.

https://material.io/tools/icons/?icon=replay&style=baseline